applications/launchpad/gui-react/src/containers/MiningContainer/MiningHeaderTip/MiningHeaderTip.test.tsx

```diff
@@ -0,0 +1,152 @@
+import { render, screen } from '@testing-library/react'
+import { Provider } from 'react-redux'
+import { configureStore } from '@reduxjs/toolkit'
+import { ThemeProvider } from 'styled-components'
+
+import MiningHeaderTip from '.'
+
+import themes from '../../../styles/themes'
+import t from '../../../locales'
+import { rootReducer } from '../../../store'
+import { initialState as miningInitialState } from '../../../store/mining/index'
+import { MiningNodesStatus } from '../../../store/mining/types'
+
+describe('MiningHeaderTip', () => {
+  it('should render "one step away"', () => {
```

Comment:
please add 'when...' to the test name, there is a specific state when this should happen, explain it in test name